server/controllers/profile.js

```diff
@@ -38,4 +38,16 @@ exports.loadProfile = asyncHandler(async (req, res, next) => {
       profile: profile,
     },
   });
-});
\ No newline at end of file
+});
+
+// @route GET /profiles/sitters
+// @desc Get user profiles
+// @access Public
+
+exports.loadProfiles = asyncHandler(async (req, res, next) => {
+  const pet_sitters = await Profile.find({ accountType: 'pet_sitter', address: { $exists: true }, price: { $exists: true } })
+    
+  res.status(200).send({ profiles: pet_sitters})
```

Comment:
I can see you're using `pet_sitters` here. Make sure to `camelCase` the variables in your schema.